site/docs/plugins/i18n.md

```diff
@@ -1,105 +1,751 @@
-# Internationalization Support (`i18n`)
+# Internationalization (`i18n`)
 
-When you want to target a multilanguage audience with your bot, you will need internationalization.
-The grammy i18n plugin helps you with that.
+The internationalization plugin makes your bot speak multiple languages.
 
-## Usage
+## Internationalization Explained
+
+> This section explains what internationalization is, why it is needed, what is complicated about it, how it relates to localization, and why you need a plugin for all of this.
+> If you think you are already fine with these, scroll right to [Getting Started](#getting-started).
```

Comment:
AFAIK _to be fine with sth_ does not mean knowing something, but rather accepting something.

Reply:
Should I revert it back to

> If you already know these things, scroll down to [get started](#getting-started).

Reply:
Reverting would work, rephrasing to something correct would also work. I'm fine with this either way.

Reply:
I'll revert it because I can't find anything better.